src/libraries/System.Text.Json/src/System/Text/Json/Serialization/MetadataPropertyName.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
```

Comment:
I don't believe any of these using statements are necessary.

Reply:
That's true, will fix.